AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -72,6 +72,11 @@ - (ASCellNode *)tableView:(ASTableView *)tableView nodeForRowAtIndexPath:(NSInde
   return nil;
 }
 
+- (ASDataControllerCellNodeBlock)tableView:(ASTableView *)tableView nodeBlockForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+  return nil;
+}
```

Comment:
Will add tests in a follow-up PR. Manually ran through use-cases to ensure that there are no obvious issues but will have to enforce this with tests in a follow up PR